### PR TITLE
Update Simplified Chinese translation

### PR DESCRIPTION
**Pull request by kxxoling**
I've updated the Chinese translation in ``zh.i18n.json`` to make it more in line with the language habits of the Chinese people. But seems not all the phrases are listed in the json file to be translated, how could I do to get them?

### CODE REVIEW

**Comment by philipwhiuk** [comment]
What's your thoughts on #2331 - clearly this affects the intended translation. Should we rename the file at the same time as 'matching language habits'.

**Comment by kxxoling** [comment]
So there's no software to pick phrases automatically? This would be a pitty.

File: i18n/zh.i18n.json [hunk 1]
```diff
@@ -2,19 +2,19 @@
...
   "Accounts_ManuallyApproveNewUsers" : "手动审核新用户",
-  "Accounts_OAuth_Facebook" : "Facebook 登入",
-  "Accounts_OAuth_Facebook_id" : "Facebook应用程序ID",
-  "Accounts_OAuth_Facebook_secret" : "Facebook Secret",
+  "Accounts_OAuth_Facebook" : "使用 Facebook 帐户登录",
+  "Accounts_OAuth_Facebook_id" : "Facebook 应用 ID",
+  "Accounts_OAuth_Facebook_secret" : "Facebook 应用 Secret",
   "Accounts_RegistrationRequired" : "需要注册",
   "Activate" : "激活",
-  "Add_Members" : "加入成员",
```
Comment (Arthur2e5) [inline i18n/zh.i18n.json]: Huh…These sound like TW. 載入、讀入、連線、登入……

**Comment by engelgabriel** [comment]
Hi, there is a software, we use https://lingohub.com

Please come to https://demo.rocket.chat/direct/gabriel.engel and send your emails to me so I can invite you to join the translation team.

**Comment by kxxoling** [comment]
@engelgabriel I've just registered lingohub, my username is ``kxxoling``.

**Comment by Arthur2e5** [comment]
`Arthur2e5` is happy to help too (when I have time, of course)

**Comment by engelgabriel** [comment]
I have now added the variations #2375

**Comment by kxxoling** [comment]
@engelgabriel Get it!

**Comment by kxxoling** [comment]
How can I dumps the translation on lingohub to JSON? Lingohub really disturbs me where the translated terms always changes the next day.

**Comment by engelgabriel** [comment]
What do you mean? Is there a bug? Ping me as soon as you finish the translation and I can dump the JSON.

**Comment by kxxoling** [comment]
I guess there may be some bug maybe caused by the fucking Chinese networking. So I'd rather translate them offline.

Some of the left terms are related to the environment, I need to know where it's used. Is there any good way?

And some of the left involved professional knowledge I'm not familiar with, need more time with them.

**Comment by engelgabriel** [comment]
You can change the JSON files directly and do a Pull Request to merge them back to the project. This will update Lingohub too.

**Pull request closed without merging**